Embedded System L1/AVR ATMEGA32/DIO/PROJECT/main.c

```diff
@@ -0,0 +1,21 @@
+
+#include"STD_TYPES.h"
+#include"BIT_MATH.h"
+#include"DIO_interface.h"
+#include "avr/delay.h"
+
+void main(void)
+{
+    ST_DIO_config_t Led ={PORTA,PIN0,OUTPUT};
+    ST_DIO_config_t Button ={PORTC,PIN0,INPUT};
+    DIO_voidSetPinDirection(&Led);
+    while (1)
+    {
+    	if(DIO_u8GetPinValue(PORTC,PIN0)==1)
+    	{
+			while(DIO_u8GetPinValue(PORTC,PIN0)==1);
+			_delay_ms(10);
+    		DIO_voidTogglePin(PORTA,PIN0);
+    	}
+    }
+}
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of c files

[Sprints_CodeGuideLine.pdf](https://github.com/KhalidMosad/Sprints-Tasks/files/6901176/Sprints_CodeGuideLine.pdf)

![4](https://user-images.githubusercontent.com/44152298/127508098-dd2316ae-1851-429c-80c2-ae418e29dd28.PNG)